src/onepay-checkout.js

```diff
@@ -666,12 +667,16 @@ function getHttpRequestInstance() {
   return new XMLHttpRequest();
 }
 
-function prepareOnepayHttpRequestParams() {
+function prepareOnepayHttpRequestParams(params) {
   let paramsUrl = 'channel=WEB';
-
-  if (typeof SmartPhone !== 'undefined' && (SmartPhone.isAndroid() || SmartPhone.isIOS())) {
+  if (typeof Smartphone !== 'undefined' && (Smartphone.isAndroid() || Smartphone.isIOS())) {
     paramsUrl = 'channel=MOBILE';
   }
+
+  paramsUrl += '&' + params.map(function (param) {
+    return encodeURIComponent(param.name) + '=' + encodeURIComponent(param.value);
+  }).join('&');
+  console.log(paramsUrl);
```

Comment:
We should remove this log statement. After that we are ready to merge I think.

Reply:
Removed